students/kulon_aleksander/lesson_05/maximum.py

```diff
@@ -0,0 +1,19 @@
+m, n = [int(s) for s in input().split()]
+max_i, max_j = m + 1, n + 1
+tab = []
+
+for i in range(m):
+    tab.append(input().split())
+    if i == 0:
+        max = int(tab[0][0])
```

Comment:
Try to avoid using built-in functions names (https://docs.python.org/3/library/functions.html)